safe/impact_functions/volcanic/volcano_polygon_building/impact_function.py

```diff
@@ -46,6 +46,9 @@ def __init__(self):
         self.volcano_names = tr('Not specified in data')
         self._target_field = 'Hazard'
 
+        # From BuildingExposureReportMixin
+        self.building_report_threshold = 25
```

Comment:
Why do you need to initialise it? Because in the `run` method, it will be replaced by the value from the postprocessor